tests/arch/arm64/arm64_high_addresses/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

```suggestion
find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
```

We use CMake packages now.